src/parser/jobs/ast/modules/Draw.tsx

```diff
@@ -79,6 +79,12 @@ export default class Draw extends Module {
 	}
 
 	private onDraw(event: CastEvent) {
+
+		// ignore precasted draws
+		if (event.timestamp < this.parser.fight.start_time) {
+			return
+		}
```

Comment:
Does ignoring it break pre-5.3, or is this a bug fix?

Reply:
This is a fix from back when we started spoofing precast stuff. Didn't catch this. It's a bug fix. Forgot to make the distinction clear sry